rest_api/sawtooth_rest_api/route_handlers.py

```diff
@@ -499,6 +503,12 @@ def _check_status_errors(proto, content, error_traps=None):
         except AttributeError:
             pass
 
+        try:
+            if content.status == proto.INVALID_SORT:
+                raise errors.SortInvalid()
+        except AttributeError:
+            pass
```

Comment:
Stylistically, I'd put comments everywhere except/pass is used so the intent is clear.  "# ok if status field not on content" or similar.  Better might be explicitly checking for it and not relying on exceptions.